core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -4,9 +4,9 @@
  * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance with the License. You may obtain a
  * copy of the License at
- * 
```

Comment:
In the current formatter, this is a space. Please adjust for that. Otherwise, there are lots of changes not related to the real logic.